docs/Team/index.md

```diff
@@ -15,88 +15,25 @@ If you're reading this page, you probably want to know more about the actual dev
 
----
-
-### Senior Software Developer
-
-**Traits**
-
-* Productive in at least two areas, ideally in more.
-* Deep understanding of one area.
-* Doesn't require supervision.
-* Mentors less experienced developers.
-* Can serve as a Subject Matter Expert in sales calls.
-* Can propose ideas, discuss, and disagree properly with clients, including stakeholders.
-* Capable of public speaking and spreading the Good Word from time to time. This one especially is not a hard & fast rule because a lot of people fear this more than dying.
-
-**Day to Day Examples**
-
-A Senior Developer can run Sprint planning and grooming meetings.
-
-They can also run a Storycarding or Feature Diagramming session, and write Technical Requirements Docs.
-
-They’ll be coding a lot, and expected to have well factored results. This code should have intelligently written tests to go along with it.
+<div id="the-ladder" class="the-ladder"></div>
 
----
-
-### Lead Software Developer
-
-**Traits**
-* Can lead a team of developers. This is different from management.
-* Can pick up a new technology and be productive fairly quickly
-* Does a lot of non-coding stuff such as:
-  * Requirements gathering
-  * Customer management
-  * Sales support
-* Manages clients, coaches them, and gets them on board with the Goal for the engagement.
-* Makes their practice better.
-* Demonstrates excellence in:
-  * Team dynamics
-  * Planning and estimating
-  * Keeping projects on the rails
-
-**Day to Day Examples**
+Read more about what these tracks look like for [Individual Contributors]({{ site.root }}/Team/ic) and [Management]({{ site.root }}/Team/management).
 
-A Lead Dev will have a high degree of ownership in their stack. They’ll be able to help write out plans for how work should be done and set goals for a group to work towards.
+### What title is not
+Title does not inflate anyone's worth in the company. We’re a relatively flat organization and that benefits us. A Lead Developer is not necessarily anyone’s boss, though they should be demonstrating leadership. In fact, everyone can (and should) be demonstrating leadership, regardless of title.
 
-Leads mentor more inexperienced developers more often. This could be answering questions, pair programming, or running code reviews. Leads help Software Developers grow to Seniors and Seniors to Leads.
+There is no pressure to gain ‘rank’ just for the sake of doing so. If you’re happy as a Software Developer, that’s great. Keep it up. However, it can be pretty fulfilling to add some more arrows to your quiver.
 
-They have more conversations with clients. Per the description, Leads will be presenting project plans to clients, getting their buy-in, and working with the PM team to keep them apprised every step of the way.
+All developers, regardless of title need to carry themselves well, have empathy for others, and demonstrate our Core Values.
```

Comment:
comma after "regardless of title":
All developers, regardless of title, need to carry themselves well...

Reply:
Maybe link to our core values web page?

Reply:
@mmislam101 yeah... the core values page got removed in the site refresh. I've been told that they'll come back at some point.